flink-doris-connector/src/main/java/org/apache/doris/flink/sink/batch/DorisBatchStreamLoad.java

```diff
@@ -128,6 +132,7 @@ public DorisBatchStreamLoad(
                         && !loadProps
                                 .getProperty(GROUP_COMMIT)
                                 .equalsIgnoreCase(GROUP_COMMIT_OFF_MODE);
+        this.enableGzCompress = loadProps.getProperty(COMPRESS_TYPE, "").equals(COMPRESS_TYPE_GZ);
```

Comment:
Do we need to check whether the current file format is `csv`? If the data here is `json`, will gzip be turned on?

Reply:
If json and compression are configured, streamload will report an error when writing.